fastlane/Fastfile

```diff
@@ -239,7 +241,71 @@ platform :android do
       end
     ensure
       sh("git reset --hard")
-      sh("git clean -fd")
+      sh("cd .. && git clean -fd")
+    end
```

Comment:
So this resets the repo at the current dir and cleans the one in the parent? Is this correct?
Also maybe you can use `git -C .. clean -fg` instead of doing cd

Reply:
This looks like it should also work 😁 I can change it to that!